server/protocols/jmap-rfc-8621/pom.xml

```diff
@@ -38,10 +38,34 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-jmap</artifactId>
+        </dependency>
+        <!-- TODO: remove this dependency-->
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-jmap-draft</artifactId>
```

Comment:
Should I extract common things between draft and RFC version into a common maven module?

Reply:
I would say it's worth trying?